test/file-translations.spec.ts

```diff
@@ -105,7 +105,9 @@ describe("SmartlingFileTranslationsApi class tests.", () => {
 
             params
                 .setFileType(FileType.XML)
-                .setFileContent(testFileContent);
+                .setFileContent(testFileContent)
+                .addDirective("entity_escaping_strategy", "propagade")
```

Comment:
I believe all directives should have `smartling.` prefix. I understand that we can put whatever we want in the test. But some times tests are a good source to figure out how it should work.

Reply:
No. Exactly in FTS there is a validation that doesn't allow `smartling.<...>` format. COnfirmed with @andy-radchenko